homeassistant/components/mill/climate.py

```diff
@@ -46,7 +54,11 @@
 )
 
 
-async def async_setup_entry(hass, entry, async_add_entities):
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: Callable[[Sequence[Entity]], None],
```

Comment:
```suggestion
    async_add_entities: Callable[[Iterable[Entity]], None],
```

Reply:
Why?

Reply:
`Sequence` is more specialised type that `Iterable`. It also should support `__getitem__` method.

Callback which is passed here expects `Iterable`:
https://github.com/home-assistant/core/blob/922eec09098e0462b64bea91210a674512b8530a/homeassistant/helpers/entity_platform.py#L309